src/hll.c

```diff
@@ -1388,7 +1416,7 @@ multiset_unpack(multiset_t * o_msp,
             }
 
             // Make sure the explicit array fits in memory.
-            if ((i_size - hdrsz) > MS_MAXDATA)
```

Comment:
Do not forget 8 bytes occupy by `ms_explicit_t::mse_nelem`!

So this should be `(i_size - hdrsz) > (MS_MAXDATA - sizeof(size_t))`!